applications/GeoMechanicsApplication/custom_processes/find_neighbour_elements_of_conditions_process.cpp

```diff
@@ -78,12 +83,12 @@ void FindNeighbourElementsOfConditionsProcess::FindNeighbouringElementsForAllBou
     }
 }
 
-void FindNeighbourElementsOfConditionsProcess::FindConditionNeighboursBasedOnBoundaryType(auto generate_boundaries)
+void FindNeighbourElementsOfConditionsProcess::FindConditionNeighboursBasedOnBoundaryType(auto GenerateBoundaries)
 {
     for (auto& r_element : mrModelPart.Elements()) {
-        const auto& rGeometryElement    = r_element.GetGeometry();
-        const auto  rBoundaryGeometries = generate_boundaries(rGeometryElement);
-        AddNeighbouringElementsToConditionsBasedOnOverlappingBoundaryGeometries(r_element, rBoundaryGeometries);
+        const auto& r_element_geometry  = r_element.GetGeometry();
+        const auto  boundary_geometries = GenerateBoundaries(r_element_geometry);
+        AddNeighbouringElementsToConditionsBasedOnOverlappingBoundaryGeometries(r_element, boundary_geometries);
```

Comment:
Since both variable are used only once and their names are very similar to the functions that initialize them, we might as well eliminate these intermediate variables, i.e.:
```suggestion
        AddNeighbouringElementsToConditionsBasedOnOverlappingBoundaryGeometries(r_element, GenerateBoundaries(r_element.GetGeometry()));
```
If this is the only review suggestion left, feel free to skip it for now.